base_local_planner/src/local_planner_limits/__init__.py

```diff
@@ -37,8 +37,11 @@ def add_generic_localplanner_params(gen):
 
     gen.add("xy_goal_tolerance", double_t, 0, "Within what maximum distance we consider the robot to be in goal", 0.1)
     gen.add("inner_xy_goal_tolerance", double_t, 0, "Best effort distance for the robot to reach the goal", 0.1)
+    gen.add("xy_min_goal_tolerance", double_t, 0, "minimum tolerance needed to reliably reach a goal; if the goal is within an obstacle, the goal can be displaced by (xy_goal_tolerance - xy_min_goal_tolerance)", 0.1)
     gen.add("yaw_goal_tolerance", double_t, 0, "Within what maximum angle difference we consider the robot to face goal direction", 0.1)
 
+    gen.add("goal_obstacle_approach_distance", double_t, 0, "In case the goal is blocked, continue moving towards it until the obstacle blocking the goal is approached is within this distance", 1.0)
```

Comment:
```suggestion
    gen.add("goal_obstacle_approach_distance", double_t, 0, "In case the goal is blocked, continue moving towards it until the obstacle blocking the goal is approached within this distance", 1.0)
```